### ISSUE TITLE
PGError: ERROR: invalid input syntax for type date: "--- !ruby/object:Date {} "

### ISSUE THREAD

**Issue by zedtux** [resque/resque#662]
I'm using Resque (1.22.0) with Rails (2.2.2) and REE.

When I use Resque, I always do something like:

``` ruby
begin
  Resque.enqueue(MyClass, arg0, arg1, arg2)
rescue Errno::ECONNREFUSED, Timeout::Error
  MyClass.perform(arg0, arg1, arg2)
end
```

With this code, I expect to have my code always executed, even if my broker is currently down (like when executing cucumber features).

Yesterday, I discovered that [Symbol are converted to String due to JSON serialization](/defunkt/resque/issues/631), but now I'm encountering another issue that is more difficult to deal with.

I have a code that do a lot of things, and create some records. When I'm using my cucumber tests, it works fine (so using directly the `perform` method).
But when I'm trying on my local machine, using workers, then it fail with the following error:

```
PGError: ERROR: invalid input syntax for type date: "--- !ruby/object:Date {}

"
```

The error occur when trying to do an INSERT with a column of type `:date`:

```
INSERT INTO "table_name" ("field1", "field2", "field3") VALUES(NULL, '2012-09-13 15:33:59.200200',  E'--- !ruby/object:Date {}

') RETURNING "id"
```

To summarize:
- When executing the code without using workers, it work fine
- When executing the same code, with same parameters, it crash on Date object

Could you please guide me to fix this ?
